docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -2,36 +2,28 @@
 To extract the configuration metadata from an application/library the _component extractor_ will be used. It extracts the list of all the config interfaces defined in a library or application. The output is a json file `component.config.metadata.json` containing an array of all the components configurations (app configs, pages, components).
```

Comment:
To extract the configuration metadata from an application/library**,** the _component extractor_ will be used (?)

Reply:
I don't think we should add a comma here 
@sdo-1A what do you think about it ?

Reply:
Here there should be a comma yes: To extract the configuration metadata from an application/library, the _component extractor_ will be used.

If we don't want a comma it would be reworded like this: The _component extractor_ will be used to extract the configuration metadata from an application/library.

Reply:
Also, the component extractor should be bold, not italic